features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
```

Comment:
@aspiers Making use of unique tags for scenarios is important for calling the particular scenarios from rake tasks. Yes, we end up with a tag for every single scenario, but that's the price. On the other hand, we are making use of it for feature specific configuration and rake tasks so far. Still it does not limit us to use other scenario tags in different contexts.